fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -7,30 +7,60 @@
 //! combined with a form of dependency injection, would provide the machines
 //! with all the unit testing they would ever need. Fate, it seems, has a sense
 //! of irony...
-
 #![no_std]
-
```

Comment:
I think we should add `#![deny(missing_docs)]` here like in the other enclave crate, and then add doc strings as required. It is really important to have good code comments in the enclave code